client/src/app/site/deployment-center/deployment-center-setup/validators/vsts-validators.ts

```diff
@@ -82,21 +82,37 @@ export class VstsValidators {
                         }/${DeploymentCenterConstants.editReleaseDefinitionPermission}?tokens=${currentProject.id}`,
                         true,
                         callHeaders
+                      ),
+                      _cacheService.get(
+                        `https://${vstsAccountValue}.visualstudio.com/${
+                          currentProject.id
+                        }/_apis/distributedtask/queues?queueNames=${DeploymentCenterConstants.agentQueueNames.join(',')}&actionFilter=${
+                          DeploymentCenterConstants.queueActionFilter
+                        }&api-version=5.1-preview.1`,
+                        true,
+                        callHeaders
                       )
                     );
                   });
               }
               return Observable.of(null);
             })
             .map(results => {
-              if (results && results.length === 2) {
+              if (results && results.length === 3) {
                 const buildPermissions = results[0] ? results[0].json().value[0] : true;
                 const releasePermissions = results[1] ? results[1].json().value[0] : true;
+                const queuePermissions = results[2] && results[2].json().count > 0 ? true : false;
```

Comment:
@bishal-pdMSFT 
We used to only use 3 queues while setting up pipeline: VS2017, ubuntu and default. 
Should we check if one of them is present in the list, else we will still fail ?